lms/static/scripts/frontend_apps/components/LMSFilePicker.tsx

```diff
@@ -259,7 +270,7 @@ export default function LMSFilePicker({
     case 'authorize':
       continueButton = (
         <AuthButton
-          authURL={/** @type {string} */ (listFilesApi.authUrl)}
+          authURL={listFilesApi.authUrl ?? ''}
```

Comment:
This should use `listFilesApi.authUrl!` to be equivalent to the previous code. If we get to this point and no `authUrl` has been provided, that is a mistake. Using a cast or `!` makes it clear what we are expecting, whereas adding a runtime `?? <fallback>` suggests there are valid cases where we expect no auth URL has been provided.

Reply:
we seem to have a linting rule forbidding this. Should I suppress it?

![image](https://user-images.githubusercontent.com/2719332/212910812-52ea02fc-f6a2-4bca-a12c-90938a7cbba4.png)

Reply:
Ah, yes. That rule should be turned off in all projects. It is part of @typescript-eslint's default ruleset. See also the client's ESLint config for some other typescript-eslint rules that we turned off.